### PR TITLE
Added recent work about me and skills

### PR DESCRIPTION
**Pull request by alexmedinasf**
index.html: Created 2 new pages, one with my recent works and see project buttons and one with an about me page and get my resume buttons
styles.css: styled everything, including buttons and images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,5 +27,126 @@ <h2>I’am a Software Developer</h2>
...
+				<section id="feature-project">
+					<img src="images/featprojects.png" alt="Feature Projects">
+					<h2>Multi-Post Stories</h2>
+					<p class="multipost">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+					<ul class="languages">
+						<li>Css</li>
+						<li>Html</li>
+						<li>Bootstrap</li>
+						<li>Ruby</li>
+					</ul>
```
Comment (vickymarz) [inline index.html:42]: - [ ] Kindly add the appropriate background color to the list of technologies and also remove the border you added to match the Figma design template chosen 👍🏼 .

| Figma Design  | Your Design |
| ------------- | ------------- |
|image  | image  |

File: index.html [hunk 2]
```diff
@@ -27,5 +27,126 @@ <h2>I’am a Software Developer</h2>
             <li><a href="#"><img src="./images/m.png" alt="M"></a></li>
 			</ul>
 			</section>
+			<main>
+				<h2 class="myrecent">My recent works</h2>
+				<hr>
+				<section id="feature-project">
+					<img src="images/featprojects.png" alt="Feature Projects">
+					<h2>Multi-Post Stories</h2>
+					<p class="multipost">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```
Comment (vickymarz) [inline index.html:36]: - [ ] Kindly add the appropriate `line-height` to the project description and decrease its` font-weight` to match the Figma design template chosen as the texts are currently too close hence difficult to read👍🏼 .

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 3]
```diff
@@ -27,5 +27,126 @@ <h2>I’am a Software Developer</h2>
...
+							<li>Html</li>
+							<li>Bootstrap</li>
+							<li>Ruby</li>
+						</ul>
+						<button class="probut" type="button">See Project</button>
+					</div>
+				</section>
+				<section class="meabout">
+					<div class="abmehead">
+						<h2>About me</h2>
```
Comment (vickymarz) [inline index.html:110]: - [ ] Kindly add the appropriate margin between the **`<section class="meabout">`** and your **About me**  title to match the Figma design chosen. Kindly increase the space as it is currently too close 👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 4]
```diff
@@ -27,5 +27,126 @@ <h2>I’am a Software Developer</h2>
...
+						<p class="abmedesc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (vickymarz) [inline index.html:111]: - [ ] Kindly add the appropriate `line-height` to the about me description and decrease its` font-weight` to match the Figma design template chosen as the texts are currently too close hence difficult to read👍🏼 .

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image)  |

File: index.html [hunk 5]
```diff
@@ -27,5 +27,126 @@ <h2>I’am a Software Developer</h2>
...
+								<li>Selenium</li>
+							</ul>
+						</div>
+						<div class="feature-item">
+							<h4><img src="./images/circle.png" alt="circle" class="icon-skill" />Skills</h4>
+							<ul>
+								<li>Codekit</li>                    
+								<li>Github</li>
+								<li>Codeopen</li>
+								<li>Gitlab</li>
```
Comment (vickymarz) [inline index.html:141]: - [ ] Kindly remove the bullet point present in all your list of technologies (You can remove them by writing `list-style-type: none` on the styling for the `<li>` tag. Add the appropriate padding to each item and also increase the fonts 😏 to match the Figma design template chosen as the texts are currently too close hence difficult to read👍🏼 .

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @alexmedinasf , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 6]
```diff
@@ -95,3 +115,197 @@ img {
...
+  align-self: flex-start;
+  margin-bottom: 120px;
+}
+
+#projects {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: 1fr;
+  padding-bottom: 120px;
+}
```
Comment (tugsem) [inline css/style.css:142]: - [ ] Optional - I highy recommend adding these borderlines around technologies and removing button outlines from orange buttons and you can add more weights to the font of these buttons: 
your design: 

image
figma design: 

image

File: css/style.css [hunk 7]
```diff
@@ -95,3 +115,197 @@ img {
...
+
+.project {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
```
Comment (tugsem) [inline css/style.css:147]: - [ ] Required - Please arrange true spacings at the sides of the every project card as the previos reviewer adressed. You can give more paddings to left and right.
your design:
image
figma design:
image

File: css/style.css [hunk 8]
```diff
@@ -95,3 +115,197 @@ img {
...
+  margin-top: 120px;
+  margin-bottom: 24px;
+  color: var(--color-accent);
+  font-size: 72px;
+  font-family: 'Crete Round', serif;
+}
+
+#aboutme hr {
+  width: 100%;
+}
```
Comment (tugsem) [inline css/style.css:204]: - [ ] Required - Kindly add more padding to match the figma design here: 

image
figma design:
image

**Review by tugsem — CHANGES_REQUESTED** [review]
Hi @alexmedinasf , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Highlights
✔️ Great work on doing requested changes
 ## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi, @alexmedinasf  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job working on the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**